### DISCUSSION TITLE
Visual Studio - Abrir Projeto ou Solução (ERRO)

### DISCUSSION THREAD

**Discussion by rmunhozdev** [balta-io/forum#136] [Tira Dúvidas] [answered]
Após criar o Projeto através de linha de comando:  dotnet new web - Blog  ..  pelo Visual Studio (Abrir Projeto ou Solução) aparece um problema ao selecionar o csproj.
O que pode ser?

![image](https://user-images.githubusercontent.com/39678569/205043730-144d7a3f-44eb-48a2-b9ca-18bb54434b8b.png)

**Comment by rmunhozdev** [comment, marked as answer]
Solucionado...
Tem que instalar esse cara aqui (não sabia) ...

![image](https://user-images.githubusercontent.com/39678569/205052445-d30add5a-a816-4609-8a8b-1cf303008528.png)

**Reply by andrebaltieri** [reply]
Obrigado por avisar @rmunhozdev 💜
